quickstart/etcdmigration.md

```diff
@@ -0,0 +1,263 @@
+---
+Title: ETCD Migration Procedure
+---
+
+By following the given steps, you can successfully migrate ETCD from one node to another during maintenance activities like node drain etc., ensuring the continuity and integrity of the ETCD data.
+
```

Comment:
we can add a note here to take a etcd snapshot like we have in legacy upgrade document https://mayastor.gitbook.io/introduction/additional-information/upgrade/legacy-upgrade